core/src/autogluon/core/trainer/abstract_trainer.py

```diff
@@ -1104,6 +1215,29 @@ def _add_model(self, model: AbstractModel, stack_name: str = 'core', level: int
                 elif level <= self.model_graph.nodes[base_model_name]['level']:
                     raise AssertionError(f"Model '{model.name}' depends on model '{base_model_name}', but '{base_model_name}' is not in a lower stack level. ('{model.name}' level: {level}, '{base_model_name}' level: {self.model_graph.nodes[base_model_name]['level']})")
                 self.model_graph.add_edge(base_model_name, model.name)
+        if model.val_score is not None:
+            if model.eval_metric.name != self.eval_metric.name:
+                logger.log(20, f'\tNote: model has different eval_metric than default.')
+            logger.log(20, f'\t{round(model.val_score, 4)}\t = Validation score   ({model.eval_metric.name})')
+        if model.fit_time is not None:
+            logger.log(20, f'\t{round(model.fit_time, 2)}s\t = Training   runtime')
+        if model.predict_time is not None:
+            logger.log(20, f'\t{round(model.predict_time, 2)}s\t = Validation runtime')
```

Comment:
refactor to separate method - this block adds lots of visual noise when reading code

Reply:
Agreed, moved to new method